src/main/java/core/basesyntax/service/report/ReportService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.report;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ReportService {
+    List<String> report(Map<String, Integer> fruitStorage);
```

Comment:
Pass fruitStorage to impl constructor
```suggestion
    List<String> createReport();
```